src/prover_calculi/superposition.ml

```diff
@@ -903,16 +1797,164 @@ module Make(Env : Env.S) : S with module Env = Env = struct
            let pp_c_s out (c,s,sc) =
              Format.fprintf out "(@[%a@ :subst %a[%d]@])" C.pp c Subst.pp s sc in
            k C.pp c C.pp new_c (Util.pp_list pp_c_s) st.demod_clauses);
+      (* Assertion against variable clashes *)
+      (* Not sure if this assertion is in place -- maybe Zip renames the vars afterwards
```

Comment:
that's a bit worrying :p 
will someone look into it?

Reply:
this assertion never gets touched ... as far as I can tell